Orange/widgets/data/owsql.py

```diff
@@ -85,10 +87,10 @@ def __init__(self):
         vbox = gui.vBox(self.controlArea, "Server", addSpace=True)
         box = gui.vBox(vbox)
 
-        self.backendmodel = BackendModel(Backend.available_backends())
+        self.backendmodels = BackendModels(Backend.available_backends())
```

Comment:
Same as above, model should remain singular. If you prefer plural, I would simply use
self.backends = BackendModel(...)

Reply:
Oh, ok I'll do that.